contributors/devel/release.md

```diff
@@ -0,0 +1,231 @@
+which lead to subsequent PRs.  The terms feature, issue, and PR may
+be used interchangeably below, but the general labeling process
+should be consistent across artifact types.
+
+---
+
+## Definitions
+
+- *issue owners*: Creator, assignees, and user who moved the issue into a release milestone.
+- *release team*: Each kubernetes release has a team doing project
+  management tasks described
+  [here](https://git.k8s.io/sig-release/release-team/README.md).  The
+  contact info for the team associated with any given release can be
+  found [here](https://git.k8s.io/sig-release/releases/).
+- *Y days*: Refers to business days (using the location local to the release-manager M-F).
+
+## The Release Cycle
+
+![Image of one kubernetes release cycle](release-cycle.png)
+
+Kubernetes releases currently happen four times per year.  The release
+process can be thought of as having three main phases:
+* feature definition
+* implementation
+* stabilization
+But in reality this is an open source and agile project, with feature
+planning and implementation happening at all times.  And given the
+project scale and globally distributed developer base, it is critical
+to project velocity that continuous integration testing insures the
+project is continuous stable so that individual commits can be
+flagged as having broken something.
+
+With ongoing feature definition through the year, some set of items
+will bubble up as targeting a given release.  The **feature freeze**
+starts ~4 weeks into release cycle.  By this point all intended
+feature work for the given release has been defined in suitable
+planning artifacts in conjunction with the Release Team's [features
+lead](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/features).
+
+Implementation and bugfixing is ongoing across the cycle, but
+culminates in a code slush and code freeze period:
+* The **code slush** starts in week ~9 of the release cycle.  The master
+  branch only accepts PRs for release milestone.  No additional feature
+  development is merged after this point.
+* The **code freeze** starts in week ~10 and continues for ~2 weeks.
+  Only critical bug fixes are accepted into the release codebase.
+
+After this point only critical bug fixes happen and all remaining
+issues are resolved before release.
+
+When the code base is sufficiently stable, the master branch re-opens for
+general development and the release is finalize from a branch.
+
+Following release, the Release Branch Manager cherry picks critical
+fixes from the master branch for a period of around 9 months, leaving
+an overlap of three release versions forward support.  Thus, each
+release is part of a broader kubernetes lifecycle:
+
+![Image of kubernetes release lifecycle spanning three releases](release-lifecycle.png)
+
+## Removal Of Items From The Milestone
+
+Before getting too far into the process for adding, please note:
```

Comment:
adding features? milestone labels? 

I like the use of "item" to stand in for issue/pr/feature.

Reply:
abstraction FTW

fixed missing words after "for adding, ..."